src/vivarium/framework/utilities.py

```diff
@@ -8,8 +8,11 @@
 """
 from importlib import import_module
 from typing import Callable
+from bdb import BdbQuit
+from loguru import logger
```

Comment:
I think we may need to think about logging choices here. Since we're using different logging libraries in different places, using loguru here I think means that the logging message for the uncaught exception won't show up for any of the libraries that configure standard logging (e.g., vivarium). I might be wrong on that but we should definitely test it. If we switch to the standard logging library, I don't think vivarium cluster tools will pick it up unless we explicitly forward the logs (I only forwarded for the worker logs)